src/node/Main.hs

```diff
@@ -87,11 +73,29 @@ baseParams loggingTag args@Args {..} = do
         , bpKademliaDump = kademliaDumpPath
         }
 
+getNodeSystemStart :: (MonadIO m) => Timestamp -> m Timestamp
```

Comment:
Looks like this does what `staticSysStart` (unsafely) does. Can we do it only here, where `unsafePerformIO` isn't necessary.